build.sbt

```
@@ -202,8 +202,16 @@ lazy val zioHttpBenchmarks = (project in file("zio-http-benchmarks"))
 
 lazy val zioHttpCli = (project in file("zio-http-cli"))
   .settings(stdSettings("zio-http-cli"))
-  .settings(libraryDependencies ++= Seq(`zio-cli`))
+  .settings(
+    libraryDependencies ++= Seq(`zio-cli`),
+    testFrameworks += new TestFramework("zio.test.sbt.ZTestFramework"),
+    libraryDependencies ++= Seq(
+      `zio-test`,
+      `zio-test-sbt`,
+    ),
+  )
   .dependsOn(zioHttp)
+  .dependsOn(zioHttpTestkit)
```

We probably only want this dependency in the tests, no?

```suggestion
  .dependsOn(zioHttpTestkit % Test)
```